micromamba/src/common_options.cpp

```diff
@@ -224,30 +224,12 @@ init_channel_parser(CLI::App* subcom)
     channel_priority.needs({ "strict_channel_priority", "no_channel_priority" });
 }
 
-void
-channels_hook(std::vector<std::string>& channels)
-{
-    auto& config = Configuration::instance();
-    bool override_channels = config.at("override_channels").value<bool>();
```

Comment:
was this line lost?

Reply:
No this isn't needed anymore. See commenty above. Also we can't use `override_channels` from the new hook location in libmamba, as the `override_channels` configurable is only defined in micromama, so doesn't exist if libmamba is used from mamba.